tests/unit/test_macros.py

```diff
@@ -17,18 +17,22 @@ def setUp(self):
         )
 
         self.config = {}
+        self.var = {}
         self.default_context = {
             "validation": mock.Mock(),
             "model": mock.Mock(),
             "exceptions": mock.Mock(),
             "config": mock.Mock(),
             "adapter": mock.Mock(),
+            "var": mock.Mock(),
             "return": lambda r: r,
         }
         self.default_context["config"].get = lambda key, default=None, **kwargs: self.config.get(
             key, default
         )
 
+        self.default_context["var"] = lambda key, default=None, **kwargs: self.var.get(key, default)
```

Comment:
Holy lambda, batman. not sure what this does here. But it's identical to line 30 apart from the `default_context` key so I suppose it's fine?

Reply:
it is fine